src/legacy-components/IssueUI/index.tsx

```diff
@@ -169,6 +169,17 @@ const IssueUI = ({ issue }: Props): JSX.Element => {
             </span>
             <AddressWithCopyUI address={issue.vaultBackingAddress} />
           </div>
+          <div className={clsx('flex', 'justify-between')}>
+            <span
+              className={clsx(
+                { 'text-interlayTextSecondaryInLightMode': process.env.REACT_APP_RELAY_CHAIN_NAME === POLKADOT },
+                { 'dark:text-kintsugiTextSecondaryInDarkMode': process.env.REACT_APP_RELAY_CHAIN_NAME === KUSAMA }
+              )}
+            >
+              Request id
+            </span>
+            <AddressWithCopyUI address={issue.id} />
```

Comment:
The `AddressWithCopyUI` component is badly named, as it's used for data other than addresses. This is a legacy component and used elsewhere in the app for id values (e.g. vault transaction tables).